model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,51 @@
+groups:
+  - id: mobile-crash-event
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash
+    attributes:
+      - id: platform
+        requirement_level: "required"
+        brief: >
+            This attribute indicates the operating system name of the device on which the crash event occurred.
+        type:
+          allow_custom_values: false
+          members:
+            - id: android
+              value: 'android'
+              brief: 'Value if crash happened on an android device.'
+            - id: ios
+              value: 'ios'
+              brief: 'Value if crash happened on an ios device.'
+      - id: foreground
+        requirement_level: "required"
+        brief: >
+          This attribute indicates whether the app was in the foreground or not during the crash.
+        type: boolean
+      - id: stacktrace
+        requirement_level: recommended
+        brief: >
+          The contents of printStackTrace() being called on the throwable
+          For android crashes
+        type: string
+      - id: thread
```

Comment:
we should rather reference the existing `thread.*` attributes here, instead:
https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/attributes.md#general-thread-attributes